app/src/common/shared/com/igalia/wolvic/ui/widgets/WindowWidget.java

```diff
@@ -1589,10 +1590,13 @@ private void hideContextMenus() {
 
     public void startDownload(@NonNull DownloadJob downloadJob, boolean showConfirmDialog) {
         if (showConfirmDialog) {
+            // As of O, the prefixes are used in their standard meanings in the SI system, so kB = 1000 bytes, MB = 1,000,000 bytes, etc.
```

Comment:
well, when I tried the patch I was surprised to see the MB values using 1000 instead of 1024 for factor conversions. This may change depending on the version.